File: beecrowd/activities/2006-Identificando_o_Chá/2006.c

#include <stdio.h>

int	main(void)
{
	int	type_tea;
	int tea_one, tea_two, tea_three, tea_four, tea_five;
	int count_number;
	int count;

	scanf("%d", &type_tea);
	scanf("%d" "%d" "%d" "%d" "%d", &tea_one, &tea_two, &tea_three, &tea_four, &tea_five);
	int teas[] = {tea_one, tea_two, tea_three, tea_four, tea_five};

	count = 0;
	count_number = 0;
	while (count <= 4)
	{
		if (teas[count] == type_tea)
			count_number++;
		count++;
	}

	printf("%d\n", count_number);
	return (0);
}

File: beecrowd/activities/2006-Identificando_o_Chá/2006.py

type_tea = input()
type_tea = int(type_tea)
tea_one, tea_two, tea_three, tea_four, tea_five = input().split(" ")
tea_one = int(tea_one)
tea_two = int(tea_two)
tea_three = int(tea_three)
tea_four = int(tea_four)
tea_five = int(tea_five)

teas = [tea_one, tea_two, tea_three, tea_four, tea_five]

count = 0
count_number = 0
while count <= 4:
	if teas[count] == type_tea:
		count_number += 1
	count += 1
print(f"{count_number}")